Plate_2_data/2.cellprofiler_ic_processing/loaddata_edit_and_ic.ipynb

```diff
@@ -0,0 +1,181 @@
+{
```

Comment:
<p>This is a suggestion for future development. </p><p> </p><p>The <code>absolute()</code> method call is good, however, if fails to check if the file exists or if it is a symlink file. Therefore, I would suggest using <code>pathlib.Path.resolve(strict=True)</code> as it inherently checks if the file exists, and it gets the true absolute source path (not absolute symlink path) </p><p> </p><p>In addition, it seems that the <code>absolute()</code> is untested at this moment <a href="https://github.com/python/cpython/blob/c4a786b8ad2a625594551ab3bc991a6831ba7688/Lib/pathlib.py#L1096-L1112" target="_blank">https://github.com/python/cpython/blob/c4a786b8ad2a625594551ab3bc991a6831ba7688/Lib/pathlib.py#L1096-L1112</a> </p><p> </p><p>I would recommend using <code>pathlib.Path.resolve()</code> as it is safer when handling file paths.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Ahh I see, so I would be able to just do this with one variable: `index_directory = index_directory.resolve()` and it returns a normal path not a symlink.

I will make this an issue to fix in this for the previous modules but I will add this for the analysis module I am making after this PR.